### PR TITLE
Added an example for a simple calculator.

### PR DESCRIPTION
**Pull request by Elektron-blip**
# Added an example calculator.

## Why? 
Just cuz I felt like it. It is kinda useful for people new the lib too ig.

- [ ] New feature
- [ ] Fix bug
- [ ] Typo
- [x] Other

## Checks
- [x] Did you use the black formatter?
- [ ] Does this requires the users to change their code?
- [x] Did you test?
- [ ] Have you updated the docs?
- [x] Can you listen to our requests?
- [ ] Did you use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

File: examples/intermediate/calculator.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# By @Elektron-blip from https://gist.github.com/Elektron-blip/08fe25bd43e896d254d311721628d54f
+import math
+import datetime
+import discord
+from discord.ext import commands
+from discord_slash import SlashCommand
+from discord_components import Button, ButtonStyle, DiscordComponents
+
+client = commands.Bot(command_prefix="-", intents=discord.Intents.all())
+slash = SlashCommand(client, sync_commands=True)
```
Comment (kiki7000) [inline examples/intermediate/calculator.py:10]: Could you just use bot.command? All the other examples use that and this requires information about `discord-py-slash-command`

File: examples/intermediate/calculator.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+@slash.slash(
+    name="calculator", description="A simple calculator. Can't do anything too complex."
+)
+async def calculator(ctx):
+    m = await ctx.send(content="Loading Calculators...")
+    expression = "None"
+    delta = datetime.datetime.utcnow() + datetime.timedelta(minutes=5)
+    e = discord.Embed(
```
Comment (kiki7000) [inline examples/intermediate/calculator.py:75]: Could you put this on `m.edit`? Not creating a variable

File: examples/intermediate/calculator.py (hunk 1 above)
Comment (Elektron-blip) [reply examples/intermediate/calculator.py]: Resolved in https://github.com/kiki7000/discord.py-components/pull/82/commits/575498989f78646870de6286ae2b1507bc1c07c2

File: examples/intermediate/calculator.py (hunk 2 above)
Comment (Elektron-blip) [reply examples/intermediate/calculator.py]: Resolved in https://github.com/kiki7000/discord.py-components/pull/82/commits/575498989f78646870de6286ae2b1507bc1c07c2

**Review by Elektron-blip — COMMENTED** [review]
It looks good to me.

**Review by kiki7000 — CHANGES_REQUESTED** [review]
Could you use Cog for this?

**Comment by Elektron-blip** [comment]
> Could you use Cog for this?

I can use cogs for it but do you want it to be a cog on the same file or do you want it in multiple files?

**Comment by kiki7000** [comment]
Oh removing the command and replacing it with cog

**Pull request closed without merging**